### PR TITLE
OSDOCS-12077 GCP BYO service account RN

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.17

Issue:
https://issues.redhat.com/browse/OSDOCS-12077

Link to docs preview:
https://82298--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-installation-and-update-gcp-service-account_release-notes

QE review:
- [x] QE has approved this change.

Also added 2 missing ID's and reworded a related release note

### CODE REVIEW

**Comment by bscott-rh** [comment]
@jianli-wei PTAL at this RN announcing GCP BYO service account for 4.17. TY

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -81,9 +82,16 @@ With the replacement of Terraform, the following permissions are required if you
 
 For more information on required permissions, see xref:../installing/installing_azure/installing-azure-account.adoc#minimum-required-permissions-ipi-azure_installing-azure-account[Required Azure permissions for installer-provisioned infrastructure].
 ====
+
+[id="ocp-4-17-installation-and-update-gcp-service-account_{context}"]
+==== Installing a cluster on {gcp-full} by using an existing service account
+
+With this update, you can install a cluster on {gcp-short} by using an existing service account, allowing you to minimize the permissions that you grant to the service account the installation program uses. You can specify this service account in the `compute.platform.gcp.serviceAccount` and `controlPlane.platform.gcp.serviceAccount` parameters in the `install-config.yaml` file. For more information, see xref:../installing/installing_gcp/installation-config-parameters-gcp.adoc#installation-configuration-parameters_installation-config-parameters-gcp[Available installation configuration parameters for GCP].
```
Comment (jianli-wei) [inline release_notes/ocp-4-17-release-notes.adoc:89]: As for the "_For more information, see..._", is it possible to also tell the link of "Required GCP permissions for user-provided service accounts" (see PR https://github.com/openshift/openshift-docs/pull/81586)? Thanks!

**Review by jianli-wei — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-17-release-notes.adoc (hunk 1 above)
Comment (bscott-rh) [reply release_notes/ocp-4-17-release-notes.adoc:89]: I should be able to add this xref once the other PR merges.

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**